static/app/views/performance/transactionSummary/tagExplorer.tsx

```diff
@@ -121,6 +121,7 @@ const getColumnsWithReplacedDuration = (
   const fieldFromFilter = filterToField[currentFilter];
   if (fieldFromFilter) {
     durationColumn.name = 'Avg Span Duration';
+    return columns;
```

Comment:
I dont think this is supposed to be here

Reply:
It is